tests/queries/1_stateful/00170_s3_cache.sql

```diff
@@ -1,17 +1,17 @@
--- Tags: no-parallel, no-random-settings, no-parallel-replicas
-
+-- Tags: no-parallel, no-random-settings, no-parallel-replicas, no-debug
+-- It is slow under debug.
```

Comment:
Looks like it slowed down after I turned `allow_prefetched_read_pool_for_remote_filesystem` on by default in CI. Let's turn it off for this test instead of `no-debug`? I assume this test became slower because there are a lot of queries with `LIMIT n` in this test, where `n` is small.

Reply:
Yes, let's try. Please open another pull request, and we will choose what PR wins...

Reply:
https://github.com/ClickHouse/ClickHouse/pull/46593